deploy/rbac-clowdapp.yml

```diff
@@ -3470,9 +3464,6 @@ parameters:
   value: '10'
 - name: IT_TOKEN_JKWS_CACHE_LIFETIME
   value: '28800'
-- name: PRINCIPAL_CLEANUP_DELETION_ENABLED
-  description: Allow or disallow cleanup job to delete principals
-  value: 'False'
 - name: PRINCIPAL_CLEANUP_DELETION_ENABLED_UMB
   description: Allow cleanup job to delete principals via messages from UMB
   value: 'False'
```

Comment:
should this default be true since this is setting it for commercial? Or do you prefer to leave it false here and set it to true in app interface?

Reply:
Yea, good question! 
I set it to true for stage and prod in app-interface, which are the only places that can use UMB. So we don't have to set it in FedRamp (which we don't have access)

And this needs your approval, which can go before this PR: https://gitlab.cee.redhat.com/service/app-interface/-/merge_requests/103147

Reply:
Awesome!! That was my next question - to make sure we set that before this was merged 👍 I LGTM the app-interface MR